src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -972,16 +972,29 @@ def customizePerJob(self, job):
 
         # Performance estimates
         if job.get('estimatedJobTime'):
-            jdl.append('+MaxWallTimeMins = %d\n' % int(job['estimatedJobTime'] / 60.0))
+            estimatedMins = int(job['estimatedJobTime'] / 60.0)
+            origCores = int(job.get('numberOfCores', 1))
+            estimatedMinsSingleCore = estimatedMins*origCores
```

Comment:
Nope - this is the number of minutes this is estimated to run on one core.  For example, if a job estimate is provided of 60 minutes on 2 cores, then `estimatedMinSingleCore` should be 120.